### ISSUE TITLE
Linux Arm64 Build support

### ISSUE THREAD

**Issue by xreal** [dunglas/frankenphp#418]
Hi Dunglas,

first if all thanks for this great project! Really a great addition for the php community!

I have a lot of Linux Arm servers and wonder if there will be binaries for the platform in the future?

Thanks

**Comment by Capevace** [comment]
+1 for arm64 binaries. 

At the moment Laravel Forges Octane integration doesn't work with Hetzner arm64 servers, so you'd have to use the docker containers and miss out on the Forge advantages.

I'm struggling to build some binaries myself and can't quite get it to work so some help/pointers here would be appreciated :)

**Comment by dunglas** [comment]
Could you try building an arm64 build by following these steps? https://frankenphp.dev/docs/static/

If it works, we could add arm64 to the build matrix.

**Comment by xreal** [comment]
I have tried both ways, the native build_static.sh does not work for me. 
My system is the following: Ubuntu 22.04.3 LTS (GNU/Linux 5.15.0-91-generic aarch64)
I attached the Server logs. Does this help? 
[build-static_log.txt](https://github.com/dunglas/frankenphp/files/13737010/build-static_log.txt)

Then i tried the Docker way. This works, after a little modification of the script. I dont know if its just my docker version or a typo in the docs.
The Docs:
`docker cp $(docker create --name static-builder dunglas/frankenphp:static-builder):/go/src/app/dist/frankenphp-linux-$(uname -m) frankenphp ; docker rm static-builder`

After this command i get the output:
`Unable to find image 'dunglas/frankenphp:static-builder' locally
static-builder: Pulling from dunglas/frankenphp
no matching manifest for linux/arm64/v8 in the manifest list entries
must specify at least one container source
Error response from daemon: No such container: static-builder`

My Docker Output at the end of the "docker buildx bake --load static-builder" command:
` => => exporting layers
 => => writing image sha256:7941b34f1062451d397d9120a0daa672d13b1f9aedd3e2672694320c7dacf2a1 
 => => naming to docker.io/dunglas/frankenphp:static-builder-dev`

So i tried:
`docker cp $(docker create --name static-builder docker.io/dunglas/frankenphp:static-builder-dev):/go/src/app/dist/frankenphp-linux-$(uname -m) frankenphp ; docker rm static-builder`

Thats does the trick:
`Successfully copied 115MB to /var/tmp/franken/frankenphp/frankenphp
`

Does anybody know if the image name has to prefix with "docker.io/" ? If so, i would create a pull requests for the docs.
